exaudfclient/base/filter_swig_code.py

```diff
@@ -81,20 +83,22 @@ def filter_swig_code(target, source):
                     output.append('  (arg1)->setString(arg2,arg3,arg4);')
                     output.append('  if (arg3) delete [] arg3;')
                     while line != '}':
-                        line = rofile.next().rstrip()
+                        line = next(rofile).rstrip()
         output.append(line)
     fd = open(str(target), 'w')
```

Comment:
I there a reason to not use "with open" in this file?

Reply:
Changed